src/ocl/convolutionocl.cpp

```diff
@@ -789,6 +794,9 @@ void ConvolutionDescriptor::FindConvFwdAlgorithm(Handle& handle,
     }
     else
     {
+        ctx.skip_solutions_that_take_long_time_to_build_and_have_narrow_coverage =
```

Comment:
Please make similar change in `FindConvBwdDataAlgorithm()`

Reply:
Please resolve this.

Reply:
Fixed.